components/url/lib.rs

```diff
@@ -169,6 +174,20 @@ impl ServoUrl {
         Ok(Self::from_url(Url::from_file_path(path)?))
     }
 
+    // https://w3c.github.io/webappsec-secure-contexts/#potentially-trustworthy-url
```

Comment:
nit: I see this hasn't been done consistently elsewhere in this file, and I do think it's better to document this as `\\\ <https://w3c.github.io/webappsec-secure-contexts/#potentially-trustworthy-url>`